nodes.yaml

```diff
@@ -60,6 +60,24 @@ nodesets:
         - "mgmt"
         - "transport1"
         - "transport2"
+  connect:
+    - role: "db"
+      cpus:   1
+      memory: 1024
+      networks:
+        - "mgmt"
+        - "service"
+    - role: "master"
+      memory: 3096
+      networks:
+        - "mgmt"
+        - "service"
+        - "public"
+    - role: "dpapp"
+      networks:
+        - "mgmt"
+        - "public"
+        - "service"
```

Comment:
master role has service as eth2 and public as eth3, but dpapp has them switched around?